Makefile

```diff
@@ -1,5 +1,14 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
 export RELEASE_BRANCH?=1-18
-export RELEASE?=1
+DEFAULT_RELEASE=$(shell cat $(BASE_DIRECTORY)/release/$(RELEASE_BRANCH)/RELEASE)
+ifneq ("$(PULL_NUMBER)","")
+	export RELEASE?=$(shell expr $(DEFAULT_RELEASE) \* 10000 + $(PULL_NUMBER))
+else
+	export RELEASE?=$(DEFAULT_RELEASE)
+endif
+ARTIFACT_BUCKET?=$(shell cat $(BASE_DIRECTORY)/release/ARTIFACT_BUCKET)
```

Comment:
Instead of having individual text files for stuff like this, does it make sense to store and read from a single file? I think we definitely want to try to be conservative with the amount of files we create and limit it potentially to the GIT_TAG for now.

Reply:
It is probably a waste of time to get the artifact bucket out of the repo, but the release numbers are a good thing. I'm thinking we'll trigger new releases just by changing those release files.

Reply:
I removed that attifact backet file. That is better handled in prow.